packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -148,8 +153,15 @@ class CollapsibleMenuView extends React.Component {
     customProps.className);
     const menuButtonClassName = cx(
       'menu-button',
-      { hidden: this.menuHidden },
+      { hidden: this.collapsedMenuAlwaysShown ? false : this.menuHidden },
```

Comment:
```suggestion
      { hidden: this.collapsedMenuAlwaysShown || this.menuHidden },
```

Reply:
Emily, thanks for the tip. I can rewrite it in short version if you like that more. However, it will be
`!this.collapsedMenuAlwaysShown && this.menuHidden`
based on the logical table as you can see in the attached screenshot, or it would completely change the behaviour.
![image](https://user-images.githubusercontent.com/60744517/102978152-5a2cc000-4504-11eb-96b6-5886bd3f0765.png)

Reply:
@epammarcell You are correct! it would need to be `!this.collapsedMenuAlwaysShown && this.menuHidden`